acousticbrainz/data/__init__.py

```diff
@@ -137,15 +137,22 @@ def count_lowlevel(mbid):
 
     return InternalServerError("Ouch!")
 
-
 def get_summary_data(mbid):
+    # Initialize the return variables at the top
+    genres = None
+    moods = None
+    other = None
+    lowlevel = None
+    highlevel = None
+
     mbid = str(mbid)
     conn = psycopg2.connect(current_app.config['PG_CONNECT'])
     cur = conn.cursor()
     try:
         cur.execute("SELECT data FROM lowlevel WHERE mbid = %s", (mbid, ))
         if not cur.rowcount:
-            raise NotFound("We do not have data for this track. Please upload it!")
+            # Returning all None as the lowelevel data not found !!
+            return lowlevel, highlevel, genres, moods, other
```

Comment:
Maybe it's better to return a dictionary there and pass it into the template? I feel like there are way too many variables returned there. This will also help get rid of initialization stuff above.